fileicons/openhab-icon-theme.json

```diff
@@ -0,0 +1,150 @@
+{
+    "iconDefinitions": {
+        "_file": {
+            "iconPath": "./images/File_16x.svg"
+        },
+        "_folder": {
+            "iconPath": "./images/Folder_16x.svg"
+        },
+        "_folder-open": {
+            "iconPath": "./images/FolderOpen_16x.svg"
+        },
+        "_file_configure": {
+            "iconPath": "./images/File_16x_Configure.png"
```

Comment:
Yes, fine for me. If we can find someone who is good in designing nice consistent svg icons would be great.